btb_phylo.py

```diff
@@ -238,15 +239,17 @@ def append_multi_fasta(s3_bucket, s3_key, outfile):
     """
     # temp directory for storing individual consensus files - deleted when function
     # returns
-    with tempfile.TemporaryDirectory() as temp_dirname:
-        consensus_filepath = os.path.join(temp_dirname, "temp.fas") 
+    consensus_filepath = consensus_path + sample['Sample'] + '.fas'
+    if os.path.exists(consensus_filepath):
```

Comment:
Can make this more concise, with `if not os.path.exists(consensus_filepath):` Then there's no need for the `pass` or `else`